src/main.rs

```diff
@@ -240,7 +238,7 @@ fn run_udp(cli_config: &CliConfig, dht_pk: PublicKey, dht_sk: &SecretKey, udp_on
     };
 
     let mut server = UdpServer::new(tx, dht_pk, dht_sk.clone());
```

Comment:
Maybe, rename `server` to `udp_server` or `dht_server` ?

Reply:
right. There is already `let mut tcp_server = TcpServer::new();`

Reply:
done